interface_config.js

```diff
@@ -271,7 +271,10 @@ var interfaceConfig = {
 
     // Allow all above example options to include a trailing comma and
     // prevent fear when commenting out the last value.
-    makeJsonParserHappy: 'even if last key had a trailing comma'
+    makeJsonParserHappy: 'even if last key had a trailing comma',
+
+    // Disable superhero features
+    DISABLE_SUPERHERO: true
```

Comment:
I think it should be ENABLE_SUPERHERO as it will make it easier to do checks later